src/Apps/WorksForYou/Components/MarketingHeader.tsx

```diff
@@ -17,47 +20,53 @@ const VIDEO_URL = `${sd.FORCE_CLOUDFRONT_URL}/videos/9172018-bn-banner-xl.mp4`
 const COLLECT_URL =
   "https://artsy.net/collect?split_test[new_collect_page]=new&acquireable=true"
 
-export const MarketingHeader: SFC = () => {
-  return (
-    <Container onClick={() => (window.location.href = COLLECT_URL)}>
-      <BorderBox
-        width="100%"
-        height="220px"
-        background={themeProps.colors.black5}
-        p={0}
-        style={{
-          overflow: "hidden",
-        }}
-      >
-        <Video src={VIDEO_URL} autoPlay loop />
-      </BorderBox>
-
-      <Spacer mb={1} />
+@track({
+  context_module: Schema.Context.BNMOBanner,
+})
+export class MarketingHeader extends Component {
+  @track((_props, _state) => ({
+    action_type: Schema.ActionType.Link,
+    destination_path: COLLECT_URL,
```

Comment:
`destination_path` was a bit unclear in the JIRA ticket

Reply:
This doesn’t need to be a function if all values are static.